core/src/main/java/io/questdb/griffin/engine/functions/groupby/StringAggGroupByFunction.java

```diff
@@ -181,9 +225,36 @@ public void setNull(MapValue mapValue) {
         mapValue.putBool(valueIndex + 1, true);
     }
 
+    @Override
+    public void setEmpty(MapValue mapValue) {
```

Comment:
`setEmpty` defaults to `setNull` which should be sufficient here unless I'm missing something